packages/@misk/core/src/features/Table/Component.tsx

```diff
@@ -8,8 +8,8 @@ export interface ITableProps {
 }
 
 export const Table = (props: ITableProps) => {
-  const { data, range = [0, data.length] } = props
-  if (data && data.length > 1) {
+  const { data, range = [0, data && data.length] } = props
```

Comment:
What does the upper end of this range evaluate to if data is `undefined` or not an Array?

Reply:
Not shown in the small diff, but in that case as with if 0 rows passed in, the Skeleton shadow component is shown instead of attempting to render data elements from the range.

tldr; range isn't used in that case.